deploy/kubernetes/overlays/examples/filter-metrics.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: apps/v1beta2 # for versions before 1.8.0 use extensions/v1beta1 [apps/v1beta2]
+kind: DaemonSet
+metadata:
+  name: cadvisor
+spec:
+  template:
+    spec:
+      containers:
+      - name: cadvisor
+        command:
+          - /usr/bin/cadvisor
+          - --disable_metrics=percpu,disk,network,tcp,udp
+          - --docker_only
+          - --logtostderr
```

Comment:
I think we should either add an optional overlay with the kubernetes default arguments:
```yaml
            - --housekeeping_interval=10s
            - --max_housekeeping_interval=15s
            - --event_storage_event_limit=default=0
            - --event_storage_age_limit=default=0
```

Reply:
done